pkg/indexed_blob/README.md

```diff
@@ -0,0 +1,15 @@
+# Indexed Blob
```

Comment:
I think we should note here that in the blob each file is compressed individually, and the start and end positions are for their compressed bytes. Depending on how we do it, but later on we may need to extend this format later with the content length and possibly the media type for better http header support for these files.